solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/subnet.yaml

```diff
@@ -12,25 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #########
-# All subnets have :
+# nonp and pbmm subnets have :
 # - logging enabled for flow logs https://cloud.google.com/vpc/docs/using-flow-logs
 # - private google access enabled https://cloud.google.com/vpc/docs/private-google-access
 #########
-# Subnet unclassified northamerica-northeast1
+# Subnet nonp-main northamerica-northeast1
 # AC-4, AC-4(21)
 apiVersion: compute.cnrm.cloud.google.com/v1beta1
 kind: ComputeSubnetwork
 metadata:
-  name: host-project-id-nane1-standard-unclassified-snet # kpt-set: ${host-project-id}-nane1-standard-unclassified-snet
```

Comment:
not a show stopper, but worth mentioning that the 'unclassified' subnet is used in this example:

https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/55301f0804c02cc177c01b2048e12e646d66c96e/examples/landing-zone-v2/tier4/managed-instance-group/instance-template.yaml#L58

Reply:
fixed